spark/src/test/scala/org/apache/spark/sql/comet/CometPlanStabilitySuite.scala

```diff
@@ -263,11 +263,13 @@ trait CometPlanStabilitySuite extends DisableAdaptiveExecutionSuite with TPCDSBa
       classLoader = Thread.currentThread().getContextClassLoader)
     // Disable char/varchar read-side handling for better performance.
     withSQLConf(
+      // Comet does not yet support DPP yet
+      // https://github.com/apache/datafusion-comet/issues/121
+      SQLConf.DYNAMIC_PARTITION_PRUNING_ENABLED.key -> "false",
```

Comment:
Should we do this only if the parquet implementation is `native_full` or `native_recordbatch` ?